astroplan/plots/tests/test_sky.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import pytest
+
+try:
+    import matplotlib
+    HAS_MATPLOTLIB = True
+except ImportError:
+    HAS_MATPLOTLIB = False
+
+
+# TODO: replace this with actual plot checks once these
+# issues are resolved:
+# https://github.com/astropy/astroplan/issues/65
+# https://github.com/astropy/astroplan/issues/74
+@pytest.mark.skipif('not HAS_MATPLOTLIB')
+@pytest.mark.mpl_image_compare
```

Comment:
Oh, that may solve my problem in photutils then. I've tried the to follow the ``wcsaxes`` example, but then stuck due to the parametrized tests.